### DISCUSSION TITLE
Multi-system training got low energy accuraccy due to large energy std

### DISCUSSION THREAD

**Discussion by Franklalalala** [deepmodeling/deepmd-kit#1220] [Q&A] [answered]
I prepared 14 systems for training. They have at least 2 Carbons in different, which induced large energy std. Going through the deepmd paper, I found that, the energy accuracy was measured with test dataset's std.
So I checked my dataset's distribution. The std is 172. The pred error is 1~2, which is roughly 1% of the std. However, the accuracy is still too low. 
My systems↓
![image](https://user-images.githubusercontent.com/42742342/137571685-2775b3fa-1bb2-4525-a5c9-a6a828a77a37.png)
The distribution↓
![image](https://user-images.githubusercontent.com/42742342/137571167-134dac27-06b8-412e-b0a9-2eb7560f1b24.png)
Is there any tricks to lower training data's std? I was recommended to replace the energy with 'formation energy', however, i don't think it's a good idea, for the energy prediction is the sum of atoms' energy, it should be linear with atom number.

**Comment by njzjz** [comment, marked as answer]
The model has already had a bias for atomic energies, which is just like the formation energy.
